app/templates/preprints/view.hbs

```diff
@@ -34,7 +38,7 @@
                 </p>
                 {{!PROJECT BUTTON}}
                 <div class="p-lg osf-box-lt" id="view_project">
-                    <span> <a type="button" class="btn btn-primary pull-right" href={{concat model.baseUrl model.project}}>View project</a> The project for this paper is available on the Open Science Framework</span>
+                    <span> <a type="button" class="btn btn-primary pull-right" href={{concat config.OSF.url model.id}}>View project</a> The project for this paper is available on the Open Science Framework</span>
```

Comment:
Instead of manually creating the URL in the page, we should be able to rely on the URLs that are already part of the API response. Eg,

`node.links.html`